readthedocs/core/templatetags/core_tags.py

```diff
@@ -112,3 +112,28 @@ def key(d, key_name):
 @register.simple_tag
 def readthedocs_version():
     return __version__
+
+
+@register.filter
+def escapejson(data, indent=None):
+    """
+    Escape JSON correctly for inclusion in Django templates
+
+    This code was mostly taken from Django's implementation
+    https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#json-script
```

Comment:
Maybe we should link to the code too https://github.com/django/django/blob/2.2.2/django/utils/html.py#L74-L92